corehq/apps/sms/templates/sms/chat.html

```diff
@@ -25,6 +25,7 @@
         <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
         <script src="{% static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
         <script type="text/javascript" src="{% static 'knockout-2.3.0-legacy/knockout.js' %}"></script>
+        <script src="{% static 'style/js/hq.helpers.js' %}"></script>
```

Comment:
now that I think about this, I'm not sure if I like the idea of introducing hq.helpers.js as a dependency here because someone can add something there (like the popover code) which would break the chat window again and not know it.  is it necessary to use `.koApplyBindings` instead of `ko.applyBindings`?